### DISCUSSION TITLE
打不开商品界面

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#43] [Q&A]
![image](https://github.com/user-attachments/assets/9a604de2-eadb-4831-b3a9-57c7570af816)

**Comment by aaronameixa** [comment]
![image](https://github.com/user-attachments/assets/2f735473-62d6-43db-9b00-2c63d358293a)
只要关闭 可用库存（显示列） 就能用了

**Comment by densen2014** [comment]
## 0922
v8.9.8
- #1652 商品列表打开"可用库存列功能"错误
